### PR TITLE
add style

### PR DESCRIPTION
**Pull request by Mucyosoda**
**Bookstore app with styling**
CSS is used to make the bookstore look awesome

### CODE REVIEW

File: src/components/Links.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
         </li>
-        <li>
-          <Link to="/categories"> Categories </Link>
+        <li className="ca">
+          <Link to="/categories"> CATEGORIES </Link>
         </li>
       </ul>
+      <span className="oval">
+        <FaUserCircle />
+      </span>
```
Comment (Emmyn5600) [inline src/components/Links.js:19]: - [ ] Kindly, refer to the attached image to work on the navbar to match with the design provided.

- Try to modify the width of the navbar
- Add margin-left to the user-icon to be on the left side.
- Change the font size of the books, categories, and user-icon

--------------------------------Yours

header-1

--------------------------------Original

header-2

File: src/index.css [hunk 2]
```diff
@@ -1,18 +1,239 @@
...
+  border: none;
+  color: #0290ff;
+  background: #fff;
+  font-size: 1rem;
+}
+
+.removebtn:hover {
+  color: #fff;
+  background-color: #0290ff;
 }
```
Comment (Emmyn5600) [inline src/index.css]: - [ ] Again, refer to the below images to match with the design provided.

- The texts has to start on the same line
- Check the font size of the original design to match with yours
- The completed percentage has to be the same as the design
- Please work on the progress, chapter, and current progress. 

-----------------------------------Original

header-3

-----------------------------------Yours

header-4

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi, @Mucyosoda  

Great work on making the project requirements:clap:
You've done well implementing the project requirements, but there are still some issues which aren't been addressed yet 
 
## STATUS: CHANGES REQUESTED :recycle:

### Changes requested.

- [ ] Check the comment under the review.

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @Mucyosoda 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**